src/SprykerEco/Yves/Heidelpay/Plugin/Router/HeidelpayRouteProviderPlugin.php

```diff
@@ -0,0 +1,217 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Yves\Heidelpay\Plugin\Router;
+
+use Spryker\Yves\Router\Plugin\RouteProvider\AbstractRouteProviderPlugin;
+use Spryker\Yves\Router\Route\RouteCollection;
+
+class HeidelpayRouteProviderPlugin extends AbstractRouteProviderPlugin
+{
+    public const HEIDELPAY_BUNDLE_NAME = 'Heidelpay';
```

Comment:
https://github.com/spryker-eco/heidelpay/pull/57#discussion_r598960838 - HEIDELPAY_BUNDLE_NAME is nor route constant, and should be protected.